testdata/v2.0/yaml/sample-petstore.out

```diff
@@ -1,13 +1,86 @@
 
```

Comment:
Please name this file vocabulary-petstore.out and restore sample-petstore.out to its original contents. I think sample-petstore.out might be used by other tests.